src/software/world/ball.h

```diff
@@ -11,6 +11,8 @@
 class Ball final
 {
    public:
+    Ball();
```

Comment:
Why did we add a default `Ball` constructor while removing those for `Field` and `World`?

Reply:
it seemed like bit of work to remove initially, but it's not so bad in the end